corehq/motech/repeaters/models.py

```diff
@@ -434,22 +435,32 @@ def retire(self):
 
     def fire_for_record(self, repeat_record):
         payload = self.get_payload(repeat_record)
+        # time how long the request takes
+        start_time = time.time()
         try:
             response = self.send_request(repeat_record, payload)
```

Comment:
Is what you did here different from putting a timing context around just this one line?

Reply:
ugh I had it in my head that the timer wouldn't stop until the exception was handled and I'm not sure why. Tested in a shell and behaves as expected, so made the change 4183245cc9211c1a460addbeffff1921e6ec1012